ui/lib/core/addon/templates/components/confirm/message.hbs

```diff
@@ -0,0 +1,41 @@
+
+  {{#maybe-in-element wormholeReference false}}
+    <div class="confirm-action-message">
+      <div class="message is-highlight">
+        <div class="message-title">
+          <Icon @glyph="alert-triangle" />
+          {{title}}
+        </div>
+        <p>
+          {{message}}
+        </p>
+      </div>
+      <div class="confirm-action-options">
+        <button
+          type="button"
+          disabled={{unless showConfirm true}}
```

Comment:
What does `true` do here?

Reply:
`true` was originally needed to say "if `showConfirm` is false, disable this button". when i didn't explicitly pass in `true` i got the error `The inline form of the `unless` helper expects two or three arguments`. 

however the conditional i added in above means that these buttons are now only rendered when `showConfirm` is true, so there's no need to disable them anymore. 🎉

Reply:
Oh nice, I've never used the inline form - thanks for explaining. `unless` never jives with my brain for some reason.